lib/Core/animation.ts

```diff
@@ -33,10 +33,10 @@ const transitionEnd = (element: Element | null) =>
   });
 
 const animationTimeout = (
-  timeoutID: ReturnType<typeof setTimeout> | undefined
+  _timeoutID: ReturnType<typeof setTimeout> | undefined
```

Comment:
There's a lot of unused params across the codebase, we could instead just turn off the rule for unused params, open to suggestions (I prefer having to be explicit)

Reply:
As someone who isn't intimately familiar with the code base, it's a one time cost to prepend a "_" to (at most) 215 places, and that prevents future unused leftovers that will confuse me.